src/control/cmd/dmg/pool.go

```diff
@@ -151,16 +151,10 @@ func (cmd *PoolCreateCmd) Execute(args []string) error {
 		}
 
 		// Extra storage space needed for metadata such as the VOS file
```

Comment:
Does this comment need to be updated to match the new code?

Reply:
- [ ] Remove deprecated comment at line 153